cli/src/commands/workspace.rs

```diff
@@ -61,10 +61,20 @@ pub(crate) struct WorkspaceAddArgs {
     /// directory.
     #[arg(long)]
     name: Option<String>,
-    /// The revision that the workspace should be created at; a new working copy
-    /// commit will be created on top of it.
+    /// A list of parent revisions for the working-copy commit of the newly
+    /// created workspace. You may specify nothing, or any number of parents.
+    ///
+    /// If no revisions are specified, the new workspace will be created, and
+    /// its working-copy commit will exist on top of the parent(s) of the
+    /// working-copy commit in the current workspace, i.e. they will share the
+    /// same parent(s).
+    ///
+    /// If any revisions are specified, the new workspace will be created, and
+    /// the new working-copy commit will be created with all these revisions as
+    /// parents, i.e. the working copy commit will be a merge commit, as if you
+    /// had used `jj new r1 r2 r3 ...`.
```

Comment:
"will be a merge commit" is only true if you pass more than one argument (assuming we don't consider the degenerate case of a single parent a merge commit, of couse)

Reply:
Alright, one more go.